src/mcts/params.h

```diff
@@ -128,7 +128,7 @@ class SearchParams {
   float GetWDLRescaleRatio() const { return kWDLRescaleParams.ratio; }
   float GetWDLRescaleDiff() const { return kWDLRescaleParams.diff; }
   float GetWDLEvalObjectivity() const { return kWDLEvalObjectivity; }
-  int GetMaxOutOfOrderEvals() const { return kMaxOutOfOrderEvals; }
+  float GetMaxOutOfOrderEvals() const { return kMaxOutOfOrderEvals; }
```

Comment:
As the meaning of the function constant has changed, it makes sense to add `Factor` to the name (e.g. GetMaxOutOfOrderEvals**Factor**())

Reply:
Done